data-manager/src/main/java/org/orbisgis/datamanager/JdbcTable.java

```diff
@@ -185,7 +179,7 @@ public ResultSetMetaData getMetaData(){
      *
      * @return The parent DataSource.
      */
-    protected JdbcDataSource getJdbcDataSource(){
+    public JdbcDataSource getJdbcDataSource(){
```

Comment:
Why is `getJdbcDataSource()` public ?

Reply:
Why not ? :-)
Declare as public offers a way to re-run a new sql query when the developer has only a ISpatialTable. ie : apply a select from with a limited set of columns